components/sup/src/manager/mod.rs

```diff
@@ -650,12 +741,7 @@ impl Manager {
     fn start_initial_services_from_watcher(&mut self) -> Result<()> {
         for service_event in self.watcher.initial_events()? {
             match service_event {
-                SpecWatcherEvent::AddService(spec) => {
```

Comment:
This was mainly a choice of where to put the complexity. If the `services` vec only has services with a `DesiredState` of `Up` then we need to change the persistence logic to do more than just iterate the `services` vec. If the "downed"services are not in the services vec, they won't get hydrated to the serialized services. We could aggregate and merge the down and up services into a new data structure that we serialize to `/services` but that seems even more complex.